.github/workflows/close_stale_issues.yml

```diff
@@ -18,7 +21,7 @@ jobs:
           only-labels: needs more info
           close-issue-message: Given that there has been no additional information added, this issue will be closed for now. Please reopen and provide additional information if you wish the Dynamo team to investigate further.
       - name: Summary
+        if: ${{ steps.close_issues.outputs.closed-issues-prs }} != '[]'
         run: |
-          echo "# Closed Issues" >> $GITHUB_STEP_SUMMARY
-          echo "" >> $GITHUB_STEP_SUMMARY
-          echo "${{ steps.close_issues.outputs.closed-issues-prs }}" >> $GITHUB_STEP_SUMMARY
+          echo '# Closed Issues' >> $GITHUB_STEP_SUMMARY
+          echo '${{ steps.close_issues.outputs.closed-issues-prs }}' | jq --raw-output '.[] | "- [#\(.number) - \(.title)](\(${{ env.GITHUB_ISSUES_URL }})/\(.number))"' >> $GITHUB_STEP_SUMMARY
```

Comment:
Can we have a comment of what this does? I guess printing out the closed issue url to log?

Reply:
Updated the PR to add comment.